inspectit-oce-core/build.gradle

```diff
@@ -43,9 +42,21 @@ dependencies {
     implementation(
             'org.springframework.boot:spring-boot:2.1.1.RELEASE',
             'org.yaml:snakeyaml:1.23',
+
+            //data validation
             'javax.validation:validation-api:2.0.0.Final',
-            'ch.qos.logback:logback-classic:1.2.3'
+            'org.hibernate.validator:hibernate-validator:6.0.2.Final',
+            'org.hibernate.validator:hibernate-validator-annotation-processor:6.0.2.Final',
+            'javax.el:javax.el-api:3.0.0',
+            'org.glassfish.web:javax.el:2.2.6',
```

Comment:
this one and previous one are not really compatible:

```
+--- javax.el:javax.el-api:3.0.0
+--- org.glassfish.web:javax.el:2.2.6
|    \--- javax.el:javax.el-api:2.2.5 -> 3.0.0
```

Reply:
I checked not what would be the dependencies of the `org.springframework.boot:spring-boot-starter-validation`, are you sure we are actually needing all those `el` stuff? shouldn't this be enough somehow?

```
compileClasspath - Compile classpath for source set 'main'.
\--- org.springframework.boot:spring-boot-starter-validation -> 2.1.1.RELEASE
     +--- org.springframework.boot:spring-boot-starter:2.1.1.RELEASE
     |    +--- org.springframework.boot:spring-boot:2.1.1.RELEASE
     |    |    +--- org.springframework:spring-core:5.1.3.RELEASE
     |    |    |    \--- org.springframework:spring-jcl:5.1.3.RELEASE
     |    |    \--- org.springframework:spring-context:5.1.3.RELEASE
     |    |         +--- org.springframework:spring-aop:5.1.3.RELEASE
     |    |         |    +--- org.springframework:spring-beans:5.1.3.RELEASE
     |    |         |    |    \--- org.springframework:spring-core:5.1.3.RELEASE (*)
     |    |         |    \--- org.springframework:spring-core:5.1.3.RELEASE (*)
     |    |         +--- org.springframework:spring-beans:5.1.3.RELEASE (*)
     |    |         +--- org.springframework:spring-core:5.1.3.RELEASE (*)
     |    |         \--- org.springframework:spring-expression:5.1.3.RELEASE
     |    |              \--- org.springframework:spring-core:5.1.3.RELEASE (*)
     |    +--- org.springframework.boot:spring-boot-autoconfigure:2.1.1.RELEASE
     |    |    \--- org.springframework.boot:spring-boot:2.1.1.RELEASE (*)
     |    +--- org.springframework.boot:spring-boot-starter-logging:2.1.1.RELEASE
     |    |    +--- ch.qos.logback:logback-classic:1.2.3
     |    |    |    +--- ch.qos.logback:logback-core:1.2.3
     |    |    |    \--- org.slf4j:slf4j-api:1.7.25
     |    |    +--- org.apache.logging.log4j:log4j-to-slf4j:2.11.1
     |    |    |    +--- org.slf4j:slf4j-api:1.7.25
     |    |    |    \--- org.apache.logging.log4j:log4j-api:2.11.1
     |    |    \--- org.slf4j:jul-to-slf4j:1.7.25
     |    |         \--- org.slf4j:slf4j-api:1.7.25
     |    +--- javax.annotation:javax.annotation-api:1.3.2
     |    +--- org.springframework:spring-core:5.1.3.RELEASE (*)
     |    \--- org.yaml:snakeyaml:1.23
     +--- org.apache.tomcat.embed:tomcat-embed-el:9.0.13
     \--- org.hibernate.validator:hibernate-validator:6.0.13.Final
          +--- javax.validation:validation-api:2.0.1.Final
          +--- org.jboss.logging:jboss-logging:3.3.2.Final
          \--- com.fasterxml:classmate:1.3.4 -> 1.4.0
```